src/Plugin/Field/FieldFormatter/AdvancedEntityReferenceFormatter.php

```diff
@@ -140,13 +189,26 @@ public static function ajaxSubmit(array $form, FormStateInterface $form_state) {
    */
   public function settingsSummary() {
     $summary = parent::settingsSummary();
-    $view_mode_delta = $this->getSetting('view_mode_delta') ?: [];
     $view_modes = $this->entityDisplayRepository->getViewModeOptions($this->getFieldSetting('target_type'));
-    foreach ($view_mode_delta as $delta => $view_mode) {
-      $summary[] = t('Delta #@delta Rendered as @mode', [
-        '@delta' => substr($delta, 10),
-        '@mode' => isset($view_modes[$view_mode_delta[$delta]]) ? $view_modes[$view_mode_delta[$delta]] : $view_mode_delta[$view_mode_delta[$delta]],
-      ]);
+
+    $override = $this->getSetting('override');
+    if ($override == 'bundle') {
+      $bundle_view_mode = $this->getSetting('bundle_view_mode') ?: [];
+      foreach ($bundle_view_mode as $bundle => $view_mode) {
+        $summary[] = t('Bundle #@bundle Rendered as @mode', [
+          '@bundle' => $bundle,
+          '@mode' => isset($view_modes[$bundle_view_mode[$bundle]]) ? $view_modes[$bundle_view_mode[$bundle]] : $bundle_view_mode[$bundle],
+        ]);
+      }
+    }
+    elseif ($override == 'delta') {
+      $view_mode_delta = $this->getSetting('view_mode_delta') ?: [];
+      foreach ($view_mode_delta as $delta => $view_mode) {
+        $summary[] = t('Delta #@delta Rendered as @mode', [
+          '@delta' => $delta,
+          '@mode' => isset($view_modes[$view_mode_delta[$delta]]) ? $view_modes[$view_mode_delta[$delta]] : $view_mode_delta[$view_mode_delta[$delta]],
```

Comment:
Replace `$bundle_view_mode[$bundle]` by `$view_mode`

Reply:
Also amount of isset() should be minified and why there no fallback in initially prepared array?

Reply:
This is only summary message, fallback is used in `viewElements()`

Reply:
I think fallback is in parent function call so I consider this as resolved